### ISSUE TITLE
webpackJsonp is undefined. Moduleraid does not load modules

### ISSUE THREAD

**Issue by arvindrajnaidu** [pedroslopez/whatsapp-web.js#571]
### Bug description
Looks like a recent change has caused webpackJsonp to be undefined. This means moduleraid's fillModuleArray function fails silently and nothing works thereafter.

### Reproduction steps
1. Just visit https://web.whatsapp.com
2. You expect Store to be populate with modules. However nothing is loaded
...

### Expected behavior 
For the Store global to be populated.

### Relevant code
if (typeof webpackJsonp === 'function') {
        moduleRaid.args.forEach((argument, index) => {
          try {
            webpackJsonp(...argument);
          } catch (err) {}
        });
      } else {
        try {
          webpackJsonp.push(moduleRaid.args[1]);
        } catch (err) {}
      }

### Environment (please complete the following information):
**WhatsApp**
 - Account type [Standard]:
 - Device - Mac
 - Whatsapp Web

**Library**
 - Browser [Chrome]:

**Comment by pedroslopez** [comment]
Thanks for submitting a great issue and filling the template, love to see this!

This has already been fixed in the latest release of my fork of moduleraid @pedroslopez/moduleraid and included in the latest release of whatsapp-eeb.js, so please update to 1.12.5 and you should be good

**Comment by arvindrajnaidu** [comment]
Okay, that response from you is gave me goosebumps. That was a crazy fast. I will upgrade and report back here.

**Comment by arvindrajnaidu** [comment]
As @pedroslopez  indicated bumping the version fixes the issue.

**Comment by feyi-tech** [comment]
This error is back.
webpackJsonp is not defined not defined. But I noticed there's an array defined with a close name, "webpackChunkwhatsapp_web_client", so I assigned webpackJsonp to webpackChunkwhatsapp_web_client, Now I get a different error, saying s is not defined. This error comes from whatsappweb's runtime webpack file.

I'm connecting whatsappweb to the latest android version that supports PairingCode
